scripts/bootstrap-aio.yml

```diff
@@ -33,6 +33,20 @@
     rpco_deploy_ceph: "{{ lookup('env', 'DEPLOY_CEPH') }}"
     rpco_deploy_hardening: "{{ lookup('env', 'DEPLOY_HARDENING') }}"
 
+- name: Enable trusty-backports
+  hosts: all
+  tasks:
+    - name: Modify sources.list
+      lineinfile:
+        dest: /etc/apt/sources.list.d/trusty-backports.list
+        create: yes
+        line: "{{ item }}"
+        state: present
+      with_items:
+        - "deb http://rackspace.clouds.archive.ubuntu.com/ubuntu/ trusty-backports main restricted universe multiverse"
```

Comment:
The url ``http://rackspace.clouds.archive.ubuntu.com/ubuntu/`` will only work inside a rackspace instance. The URL is not universal. Also, ``main`` and ``multiverse`` are unnecessary.